CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+
+For testing, you can use the `workflows-nextflow-dev` AWS account that was set up [here](https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/3dfe3fe2db327bd07cf31610cd77f02c3bacc130/org-formation/organization.yaml#L316-L326). You can [open an issue](https://github.com/Sage-Bionetworks-Workflows/aws-workflows-nextflow-infra/issues/new/choose) to request for access to the AWS account. One of the project admins will create an IAM user in this AWS account and share the credentials in a secure way.
+
+```
+# Install dependencies in isolated virtual environment
+pipenv install --dev
+
+# Install pre-commit hooks into Git
+pipenv run pre-commit install
+
+# Set up an AWS CLI profile with admin access for a test account
+aws configure --profile $AWS_PROFILE
+```
+
+## Testing sceptre deployment
+
+If your text editor (_e.g._ Visual Studio Code) or shell (_e.g._ using [`direnv`](https://direnv.net/)) can automatically activate the `pipenv` virtual environment, you can omit the `pipenv shell` command.
+
+```
+# Activate the pipenv virtual environment to use sceptre
+pipenv shell
+
+# Test the deployment of the 'prod' stack group
+sceptre --var "profile=$AWS_PROFILE" launch --yes prod
+
+# Delete the test deployment of the 'prod' stack group
+sceptre --var "profile=$AWS_PROFILE" delete --yes prod
+```
+
+## How this repository was set up
+
+1. Create a PR on the `organizations-infra` [repository](https://github.com/Sage-Bionetworks-IT/organizations-infra) to create an AWS account using `org-formation`.
+   - This was done twice in this case so we have both a more restricted production account and a sandbox-like development account.
+   - Here are the PRs for the [production account](https://github.com/Sage-Bionetworks-IT/organizations-infra/pull/130) and the [development account](https://github.com/Sage-Bionetworks-IT/organizations-infra/pull/131).
+2. Create a PR on the `organizations-infra` [repository](https://github.com/Sage-Bionetworks-IT/organizations-infra) to set up access to the above AWS account via JumpCloud.
+   - Once again, this was done twice, once per account.
+   - Here are the PRs for the [production account](https://github.com/Sage-Bionetworks-IT/organizations-infra/pull/135) and the [development account](https://github.com/Sage-Bionetworks-IT/organizations-infra/pull/136).
+   - Note that the PRs include an XML file that needs to be manually generated by the IT team. You can create the PR with a placeholder XML file and they will provide the actual file once they see the PR.
+   - Once these PRs are merged, the corresponding accounts appeared under Applications in [JumpCloud](https://console.jumpcloud.com/).
+3. Create a PR on the `organizations-infra` [repository](https://github.com/Sage-Bionetworks-IT/organizations-infra) to create an AWS service account using `org-formation`.
+   - In this context, "service account" doesn't refer to an AWS account.
```

Comment:
what does `doesn't refer to an AWS account`  mean?  i think we are creating a `service account`, aren't we?

Reply:
My intention was to clarify the difference between an AWS account in the technical sense (like the one created in [this PR](https://github.com/Sage-Bionetworks-IT/organizations-infra/pull/130)) and a service "account" that represents an IAM user and role within an AWS account (like the one created in [this PR](https://github.com/Sage-Bionetworks-IT/organizations-infra/pull/141)). I clarified the language a bit [here](https://sagebionetworks.jira.com/wiki/spaces/IT/pages/2058878986).